index.html

```diff
@@ -35,5 +35,149 @@ <h1>Norbert Siko</h1>
+          <button type="button">
+            See this project
+            <img src="assets/ic_arrow_right.svg" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj4.png" alt="project 4" />
+        <div class="description">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/ic_arrow_right.svg" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj5.png" alt="project 5" />
+        <div class="description">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/ic_arrow_right.svg" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj6.png" alt="project 6" />
+        <div class="description">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/ic_arrow_right.svg" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <h1>About</h1>
+      <p>
+        I can help you build a product , feature or <br />
+        website Look through some of my work <br />
+        and experience! If you like what you see <br />
+        and have a project you need coded, <br />
+        don’t hesitate to contact me.
+      </p>
+      <button type="button">
+        Get my CV <img src="assets/ic_download.svg" />
+      </button>
+      <div class="languages">
+        <h1>Languages</h1>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+        <img src="assets/hashtag.png" alt="hashtag" />
+      </div>
+      <div class="frameworks">
+        <h1>Frameworks</h1>
+        <ul>
+          <li>React.js</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capibara</li>
+        </ul>
+        <img src="assets/ic_framework.png" alt="frameworks icon" />
+      </div>
+      <div class="skills">
+        <h1>Skills</h1>
+        <ul>
+          <li>Database Management</li>
+          <li>CLI</li>
+          <li>API Design</li>
+          <li>Version Control</li>
+          <li>Web Development</li>
+        </ul>
+        <img src="assets/ic_skills.png" alt="skills icon" />
+      </div>
```

Comment:
> - [ ] I suggest you use a background image for lines **158**, **168**, and **179**, rather than using the `img `tag.  Background images are typically more lightweight and load faster than `img `tags, which can improve the performance of your website.☺

> - [ ] I suggest you wrap the technology categories ("Languages", "Frameworks" and "Skills") in a `ul` and each of them in a `li`. For example👇
> ```
> <ul>
>   <li>Languages
>     <ul>
>       <li>JavaScript</li>
>       <li>Python</li>
>       <li>Java</li>
>     </ul>
>   </li>
>   <li>Frameworks
>     <ul>
>       <li>JavaScript</li>
>       <li>Python</li>
>       <li>Java</li>
>     </ul>
>   </li>
> </ul>
```